src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -194,17 +194,21 @@ trait WorkspaceComponent {
     }
 
     def deleteWorkspaceEntitiesAndAttributes(workspaceId: UUID) = {
```

Comment:
👍 for consistency on these 3 but now I am not sure about the naming of these.  It reads like "delete entity and then delete entity".  Hmmm.